All possible fixes and updates since Nov 2018...

ivankokan opened the pull request:
Hi @maxgalbu, are you still following what's going on here?

Resolves #56.
Resolves #79.
Resolves #83.
Resolves #87.
Resolves #88.
Resolves #89.
Resolves #90.
Resolves #95.
Resolves #96.
Resolves #97.
Resolves #99.
Resolves #103.
Resolves #104.
Resolves #106.
Resolves #107.
Resolves #108.

ivankokan committed:
Merge pull request #1 from maxgalbu/master

Merge upstream

ivankokan committed:
getGameUrlWithBitrate renamed, new plain getGameUrlWithBitrate

ivankokan committed:
mpd target bitrates are added, old video heights are preserved with reasonable mappings, default quality set to max

ivankokan committed:
Explicit version 1.1.7 for script.module.pil

ivankokan committed:
Error handling when generating thumbnail

ivankokan committed:
Games from previous seasons are no longer included, Readme.md update

ivankokan committed:
Fallback to original thumbnail of home team

joshjryan commented:
Hi @maxgalbu,
I have been using @ivankokan's branch for the past two weeks, and it successfully handles the exception being thrown when loading thumbnails on the Fire TV (falling back to old functionality of just displaying the home team, instead of creating an image composed of both team logos).

joshjryan commented:
Hi @maxgalbu,
With this merge request from @ivankokan's fork, and the new fixes that have gone into inputstream.adaptive (available in the latest xbmc nighly), this plugin now works on Amazon FireTV Sticks.
It'd be great if we could get this change merged into your official repo, and get it updated in the official Kodi addon repository (where an old version of this plugin is currently listed).
Let me know if there is anything I can do to help.

joshjryan commented:
@ivankokan, because it has been 10 months since @maxgalbu's last commit, do you think we should assume he is no longer supporting this plugin? (if that's the case, @maxgalbu, thank you for all of your work to get this plugin where it is!)

If so, @ivankokan, would you be interested in submitting your repo to kodi to be added to the official repo?
*I don't know if there is a process defined to update https://kodi.tv/addon/plugins-video-add-ons/nba-league-pass to point to your repo.

ivankokan commented:
Hi @joshjryan, let's wait a bit more for @maxgalbu's answer before making any decision.

ivankokan commented:
Hi @maxgalbu, are you still following what's going on here?

maxgalbu commented:
Sorry, I'm no longer following this plugin, do you have the time to take care of it?

ivankokan commented:
> Sorry, I'm no longer following this plugin, do you have the time to take care of it?

It is better that someone takes care than no one - I can give a shot.

ivankokan commented:
Hi @maxgalbu, how do we "transfer" everything?

ivankokan commented:
Hi @maxgalbu. I see that the repository can be completely transferred through Settings (https://help.github.com/en/articles/transferring-a-repository). On the other hand, do we need to change things somewhere else within the Kodi community?

ivankokan commented:
Hi @maxgalbu, any progress?

ivankokan commented:
Hi @maxgalbu, it would be great if we welcome the new NBA season ready with all updates, fixes, and changes (I guess there are many to be covered as we had streaming issues at the end of last season). How do we "transfer" everything?

ivankokan commented:
@joshjryan, have you tried the preseason, anything critical?

joshjryan commented:
The Nuggets don't start playing games until next week :)
I'll try one of the other games over the next day or two and let you know how it goes.

ivankokan committed:
Video quality removed from Settings

ivankokan committed:
Updated links

ivankokan committed:
Updated links

ivankokan committed:
Updated links

ivankokan committed:
Updated links

ivankokan committed:
Updated links

ivankokan committed:
Merge pull request #2 from ivankokan/dev

Video quality removed, updated links

ivankokan commented:
@maxgalbu Any updates?

ivankokan committed:
Pull in inputstreamhelper (pip install -r requirements.txt)

https://github.com/emilsvennesson/script.module.inputstreamhelper/issues/212

ivankokan committed:
Local importing of inputstreamhelper.Helper

https://github.com/emilsvennesson/script.module.inputstreamhelper/wiki/Integration

ivankokan committed:
Setup script

ivankokan committed:
Small refactoring

ivankokan committed:
Bug fix

ivankokan committed:
Small body and headers changes for games and Live TV

ivankokan committed:
getGameUrlWithBitrate removed

ivankokan commented:
@maxgalbu @joshjryan What do you suggest on the general matter add-on's future?

ivankokan commented:
@maxgalbu @joshjryan What do you suggest on the general matter add-on's future?

I am ready to take over this one.

ivankokan committed:
Handle unknown game cameras

ivankokan committed:
Merge pull request #3 from ivankokan/dev

Handle unknown game cameras

ivankokan committed:
LiveTV minor edit

ivankokan committed:
LiveTV clean-up

ivankokan committed:
Align getEpisodeUrl with getLiveUrl

ivankokan committed:
Improve playLive and playEpisode in LiveTV

ivankokan committed:
Minor edit

ivankokan committed:
Minor refactoring

ivankokan committed:
Minor text edit

ivankokan committed:
Formatting

ivankokan committed:
Refactor getLiveUrl

ivankokan committed:
Introduce common getPlayableItem and use it for TV as well

ivankokan committed:
Merge pull request #4 from ivankokan/dev

Fix NBA TV Live

ivankokan commented:
Resolves #99 and #103.

ivankokan commented:
I linked all relevant issues so they eventually get automatically closed upon merging this PR.

ivankokan committed:
Simplify fetching JSON

ivankokan committed:
Refactor getEpisodeUrl

ivankokan committed:
Use snake_case naming style

ivankokan committed:
Minor refactoring

ivankokan committed:
Merge pull request #5 from ivankokan/dev

Fix NBA TV Episodes

ivankokan commented:
NBA TV Live and Episodes are fixed now. It remains to sort out some ambiguities with timestamps.

ivankokan committed:
fetch_json -> fetch

ivankokan committed:
Refactor some modes

ivankokan committed:
Minor edits

ivankokan committed:
Minor edits

ivankokan committed:
Fix NBA TV episode list and its datetime operations

ivankokan committed:
Fix tzname calls

ivankokan committed:
Merge pull request #6 from ivankokan/dev

Fix NBA TV episode list and its datetime operations

ivankokan committed:
Enable content lookup

ivankokan committed:
Add pytz==2014.2 requirement

ivankokan committed:
Minor edits

ivankokan committed:
Uniformly use (old and deprecated but explicit) US/Eastern timezone

ivankokan committed:
Add python-dateutil==2.8.1 requirement

larssb commented:
How lovely it would be to get you @ivankokan to take this over. @maxgalbu would you be able to spend the time it requires to transfer the "keys" to Mr. Ivan Kokan. Please.

Thank you.

ivankokan committed:
Add more requirements

ivankokan committed:
Change License server and refactor usage of InputStream Helper

ivankokan committed:
Merge pull request #7 from ivankokan/dev

Change License server for the Season restart

ivankokan committed:
Set MIME type and disable content lookup

ivankokan committed:
TODO remarks

ivankokan committed:
Minor edit

ivankokan committed:
Refactor get_playable_item

ivankokan committed:
Refactor get_episode_url and get_live_url

ivankokan committed:
Minor edits

ivankokan committed:
Minor edits

ivankokan committed:
Refactor getGameUrl and playGame

ivankokan committed:
Import script.module.inputstreamhelper v0.5.0

ivankokan committed:
TODO remarks on Network.MacAddress

ivankokan committed:
Format source code

ivankokan committed:
Refactor SharedData

ivankokan committed:
Refactor SharedData

ivankokan committed:
Refactor SharedData

ivankokan committed:
Refactor SharedData

ivankokan committed:
Minor edits

ivankokan committed:
Revert manifest_update_parameter=full

ivankokan committed:
Merge pull request #8 from ivankokan/dev

Fix live streams (revert manifest_update_parameter=full) and other improvements

ivankokan committed:
Temporarily skip and log games without visitor/host info

ivankokan committed:
Add team season record if available

ivankokan committed:
Simplify code

ivankokan committed:
Introduce advanced processing of game keys

ivankokan committed:
login -> authenticate

ivankokan committed:
Remove force_login

ivankokan committed:
username -> e-mail

ivankokan committed:
Change authenticate return type

ivankokan committed:
Fix processed_keys initialization

ivankokan committed:
New authentication

ivankokan committed:
Refactor common.authenticate

ivankokan committed:
Refactor authenticate

ivankokan committed:
Change cookies and access_token default values

ivankokan committed:
Change publish_endpoint

ivankokan committed:
Use access_token for Games and NBA TV

ivankokan committed:
Merge pull request #9 from ivankokan/dev

New authentication for the 2020-21 NBA season

ivankokan commented:
@maxgalbu Please?

> Hi @maxgalbu. I see that the repository can be completely transferred through Settings (https://help.github.com/en/articles/transferring-a-repository). On the other hand, do we need to change things somewhere else within the Kodi community?

nre-ableton committed:
Don't show team records if show_scores disabled

We should treat "show scores" as a cover-all "no spoilers" option. To
that end, team records should not be shown if "show scores" is off, as
this can accidentally reveal game results.

nre-ableton committed:
Handle HLS protocol used for live games

Thanks to @panpawel88 for this patch!

ivankokan committed:
Merge pull request #10 from nre-ableton/nre/master/no-spoilers-in-records

Don't show team records if show_scores disabled

ivankokan committed:
Unified setting Show records and scores

ivankokan committed:
Merge pull request #11 from nre-ableton/nre/master/fix-live-games

Handle HLS protocol used for live games

ivankokan committed:
Refactor and enrich media streaming protocols

ivankokan committed:
Merge pull request #12 from ivankokan/dev

Refactor and enrich media streaming protocols
